src/runtime/components/nuxt-img.ts

```diff
@@ -41,6 +42,16 @@ export default defineComponent({
       if (props.sizes && (!props.placeholder || placeholderLoaded.value)) {
         attrs.sizes = sizes.value.sizes
         attrs.srcset = sizes.value.srcset
+      } else if (props.densities || $img.options.densities) {
+        attrs.srcset = $img.getDensitySet(props.src!, {
+          ..._base.options.value,
+          densities: props.densities,
```

Comment:
should this be:
```suggestion
          densities: props.densities || $img.options.densities,
```

Reply:
I think this should already be handled here?
https://github.com/nuxt/image/pull/769/files#diff-6b00e0c4d4cf58d68124cc022436a52ea9dd527851ad8a1397cfa336b2975a4fR229